src/plugins/filter.ts

```
@@ -23,6 +33,319 @@ const BURST_RATE_SCALE = 4 // 突发活跃斜率
+        .match(/^(\d{4})\/(\d{2})\/(\d{2})-(\d{2}):(\d{2}):(\d{2})$/)
+    if (!matched) return null
+
+    const [
+        ,
+        rawYear,
+        rawMonth,
+        rawDay,
+        rawHour,
+        rawMinute,
+        rawSecond
+    ] = matched
```

ESLint

[error] Replace `⏎········,⏎········rawYear,⏎········rawMonth,⏎········rawDay,⏎········rawHour,⏎········rawMinute,⏎········rawSecond⏎····` with `,·rawYear,·rawMonth,·rawDay,·rawHour,·rawMinute,·rawSecond`

(prettier/prettier)